docs/security/credential-vending.md

```diff
@@ -132,12 +132,19 @@ Gravitino supports custom credentials, you can implement the `org.apache.graviti
 
 Besides setting credentials related configuration, please download Gravitino cloud bundle jar and place it in the classpath of Iceberg REST server or Hadoop catalog.
 
-Gravitino cloud bundle jar:
+For Hadoop catalog, please use Gravitino cloud bundle jar with hadoop and cloud packages:
 
-- [Gravitino AWS bundle jar](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-aws-bundle)
-- [Gravitino Aliyun bundle jar](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-aliyun-bundle)
-- [Gravitino GCP bundle jar](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-gcp-bundle)
-- [Gravitino Azure bundle jar](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-azure-bundle)
+- [Gravitino AWS bundle jar with hadoop and cloud packages](https://mvnrepository.com/artifact/org.apache.gravitino/gravitino-aws-bundle)
```

Comment:
When using hadoop, we'd better use `Hadoop` except the file name or path name.

Reply:
fixed